active/online-community.md

```diff
@@ -0,0 +1,90 @@
+
+- What actions are you proposing the WG be allowed to take directly?
+  - Creation and management of processes related to community platforms. Examples of these processes include
+    - Proposals for adding/removing moderators & other roles and the adding/removing of roles directly
+    - Training for moderators & other roles as required
+    - Ensuring moderation is consistent across teams and platforms
+  - Taking ideas from the community to improve the community spaces.
+- Which actions will the WG take back to the Board for votes?
+  - Proposal to trial a new platforms
+  - Proposal to migrate to a new platforms
+  - Proposal to retire an existing platform that is in use
+  - Budget to improve existing platforms
+
+There is a lot of potential ideas in this space. Part of the WG will be setting up a process to take ideas and have a backlog of these ideas.
+
+### Delegated responsibilities
+
+With regards to Django Software Foundation responsibilities and resources, the group operates with:
+
+- A minimum of two members having admin privileges for a single platform while a single member of the group not having admin priveleges to all platforms.
+
+## Initial membership
+
+- Chair: Andrew Miller
+- Co-Chair: TBD
+- Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): Tom Carrick?
+- Other members:
+  - Patryk Bratkowski
+
+## Future membership
+
+- Who is eligible to join? Any volunteer, or are there specific requirements?
+  - Anyone can join, however they need to active on the community platforms for at least 3 months or nominated by a DSF Individual member.
+- How do people who want to join sign up / volunteer / express interest?
+  - Send one of the WG members a message on Discord or the Forum
+- How will decisions on adding/removing members be handled?
+  - New members may self-nominate.
+  - Members will serve for a one year term. At the end of this term, they need to opt into staying involved to keep being a member of the group.
+  - New members will get approved by existing members of the WG.
+  - If any member wishes to leave the group before the end of their term, they can do so by notifying the group.
+  - Members can propose a vote on removing a member from the working group. This needs 50%+1 agreement.
+
+## Budget
+
+- How much money or spending discretion do you need?
+  - None to begin with, however I would expect over time budget might get requested for supporting the Forum and Discord server. See below for [ideas](#budget-ideas)
+- How do you want that money allocated:
+  - None to begin with.
+
+## Comms
+
+Where will discussions and activities take place?
+
+Suggestions:
+
+- Private Channel in the Discord Server and private category in the forum, possibly also a channel is the DSF slack.
+- This space should be shared with the CoC Commitee or another space for shared comms across platforms.
+
+Meetings: Quarterly video call meeting on Discord.
```

Comment:
Could we not stick with Discord in the WG definitions? We could be less specific saying:
```suggestion
Meetings: Quarterly meetings, preferably over video to improve the engagement, facilitate better communication, and ensure more meaningful discussions.
```

Reply:
How the mention of Discord any different from the mention of Slack or Google Groups in other WG definitions? 

I'm happy to be corrected, but I see this repo as 'living' so if we do change how we do calls as a WG then this document can be updated to reflect that?

Reply:
At the moment the WGs (all? mostly?) have a space on Slack. I don't see this as strictly necessary as long as all WG members are happy to use the stated platform.

Reply:
See our notes on [Changes to Working Groups](https://github.com/django/dsf-working-groups?tab=readme-ov-file#changes-to-working-groups) and what qualifies as a "substantive" change that requires board approval, and what can be done without that. If in doubt just check with us :)